.ci_support/linux_64_.yaml

```diff
@@ -1,7 +1,7 @@
 cdt_name:
 - conda
 channel_sources:
-- conda-forge/label/openff-interchange_rc,conda-forge
```

Comment:
This got removed in a re-render, which should be okay if we don't need the rc to build this package